src/style/badges/badges.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import './badges.css';
+import { colors } from '../colors';
+import { components } from '../typography/typography';
+
+interface BadgeProps {
+  icon: React.ReactNode;
+  text: string;
+}
+
+export const fontBadges = {
+  fontFamily: components.family.primary,
+  fontSize: components.fontSize.large,
+}
+
+export const Badge = ({ icon, text }: BadgeProps) => {
+  const [selected, setSelected] = useState(false);
```

Comment:
o ideal seria o componente pai controlar o estado de selecionado. Perceba que no figma, esses botoes seriam usados para motificar o tipo de visualizacao da imagem na tela de detalhes. Se um badge for selecionado, todos os outros deve ser atualizados para o estado de _deselecionados_. Sendo assim, a constante `selected` e a funcao que lida com o evento de clique dentro deste botao devem ser passadas por props

Reply:
corrigi